pkg/customresource/actionset.yaml

```diff
@@ -266,6 +268,10 @@ spec:
       type: string
       description: Progress of completion in percentage
       jsonPath: .status.progress.percentCompleted
+    - name: RunningPhase
+      type: string
+      description: The phase that is being run, if the actionset is completed it would be set to ""
```

Comment:
```suggestion
      description: The phase that is being run, if the actionset is failed or completed it would be set to ""
```

Reply:
I knowingly put completed there, because it would cover failed or success both. Other wide we can write 
```
actionset is failed or succeeded it would be set to ""
```

Reply:
makes sense